src/runtimes/node/utils/zip.ts

```diff
@@ -75,11 +75,11 @@ const createDirectory = async function ({
   const functionFolder = join(destFolder, basename(filename, extension))
 
   // Deleting the functions directory in case it exists before creating it.
-  await deleteFiles(functionFolder, { force: true })
-  await fs.mkdir(functionFolder, { recursive: true })
+  await rm(functionFolder, { recursive: true, force: true, maxRetries: 3 })
```

Comment:
use rm instead of `del` package. retries on common windows errors like EPERM, EBUSY, etc. with linear backoff with 100ms more each time.